### PR TITLE
Improve reoperation modelling use to use causedBy

### PR DESCRIPTION
**Pull request by vadi2**
Improve reoperation modelling use to use causedBy instead of partOf

### CODE REVIEW

**Comment by MariekeSpanFurore** [comment]
I approve. However, I started another pull request with updates on the reoperation profile. What is most practical to do here?

**Comment by vadi2** [comment]
We need to wait on ICHOM approval to merge it - let's proceed with this one and when the time comes, we can update the other one too.

**Review by MariekeSpanFurore — APPROVED** [review]

**Pull request merged**
